README.md

```diff
@@ -1,7 +1,7 @@
-CefSharp.MinimalExample
-=======================
+# CefSharp.MinimalExample
 
-Minimal example of how the CefSharp library can be used using the official `Nuget` packages. 
+Minimal example of how the CefSharp library can be used using the official `Nuget` packages with .NET Framework
+projects (`CefSharp.MnimalExample.sln`) and .NET Core projects (`CefSharp.MinimalExample.netcore.sln`). 
```

Comment:
Small typo `MnimalExample` should be `MinimalExample`

Reply:
Thanks, fixed.